stories/2023-02-01-Overwatch-League/OWL_story.ipynb

```diff
@@ -0,0 +1,3730 @@
+{
```

Comment:
<p>Concerning: "how far the payload (an object fought over on some map types) progressed". This is a key Overwatch game style that may not be obvious to all readers. It might be worth explaining a little more about the different win conditions (not necessarily right here, but in the intorductory sections, say?).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Added some clarification.</p>